Base/usr/share/man/man1/sh.md

```diff
@@ -0,0 +1,34 @@
+## Name
+
+`sh` - shell, the command language interpreter
```

Comment:
The real name of the shell is `Shell`.
`sh` is a POSIX program that you've elected not to pursue compatibility with.
That's fine, but let's not pretend to be an `sh` since that will break the vast majority of scripts that may try to run with `#!/bin/sh`

Reply:
So what would you suggest we do (setting the name in the manpage aside)?
make Shell run `/bin/bash` if invoked as `sh`?

Reply:
I think we should probably stop calling it `/bin/sh` altogether and just be `/bin/Shell`.

Reply:
That has quite a few implications, we'd need to make `system()` and co use `/bin/Shell` for one (since now there's no `/bin/sh`).
Perhaps we could add a minimal sh interpreter and use that if the shell is invoked as `sh`?

Reply:
Yes, that would also be fine. My point is just that a non-`sh` shell should not pretend to be a `sh` :)

Reply:
Re `sh` discussions: I'll point out that ninja uses `/bin/sh -c ...` to spawn subprocesses, and it assumes that that does something sensible (which it currently mostly does). So there's some value in something sh-like existing at /bin/sh.

Reply:
I'm gonna go ahead and say that this 'sh' issue is outside the scope of _this_ PR, so I'll refer to the Shell as `Shell` in the manpages, and we'll see where we go with `/bin/sh` (perhaps as a separate issue).

Reply:
That's totally fine. And I agree with @nico there.